CDCgov/phinvads-go
# Local Docker Compose development setup

## Summary

This PR adds:

- Minor `docker-compose.yml` changes
- Updates README to account for the changes in this PR
- Adds a couple of new makefile commands to help streamline running the app and database container

## How to Test

1. Run `make startdb` to start up the database
2. Run `make refreshdb` to load in the dump file data (if needed)
3. Run `make startapp`, which will start up both the `air` and `templ` commands simultaneously
4. Check code modifications to the code are automatically triggering rebuilds/live reloading

Note: you'll need to use the proxy address ([http://localhost:7331](http://localhost:7331)) to see browser live reloading in action.
